common/src/main/java/org/apache/gravitino/dto/requests/ModelVersionUpdateRequest.java

```diff
@@ -167,4 +170,36 @@ public void validate() throws IllegalArgumentException {
           StringUtils.isNotBlank(newUri), "\"newUri\" field is required and cannot be empty");
     }
   }
+
+  /** Request to update the aliases of a model version. */
+  @EqualsAndHashCode
+  @AllArgsConstructor
+  @NoArgsConstructor(force = true)
+  @ToString
+  @Getter
+  class UpdateModelVersionAliasesRequest implements ModelVersionUpdateRequest {
+    @JsonProperty("aliasesToAdd")
+    private final String[] aliasesToAdd;
+
+    @JsonProperty("aliasesToRemove")
+    private final String[] aliasesToRemove;
+
+    /** {@inheritDoc} */
+    @Override
+    public ModelVersionChange modelVersionChange() {
+      return ModelVersionChange.updateAliases(aliasesToAdd, aliasesToRemove);
+    }
+
+    /**
+     * Validates the request
+     *
+     * @throws IllegalArgumentException If the request is invalid, this exception is thrown.
+     */
+    @Override
+    public void validate() throws IllegalArgumentException {
+      Preconditions.checkNotNull(aliasesToAdd, "aliasesToAdd field is required and cannot be null");
+      Preconditions.checkNotNull(
+          aliasesToRemove, "aliasesToRemove field is required and cannot be null");
```

Comment:
I think the checking condition for Java is inconsistent with Python one. Also, I think either `aliasesToAdd` or `aliasesToRemove` can be null, right?